german/the-black-hack.md

```diff
@@ -113,11 +113,11 @@ Zeit im Spiel wird vor allem auf zwei Arten nachgehalten - **Momente** (Runden)
 
 ## Bewegung und Entfernung
 
-Anstatt genaue Werte nachzuhalten, benutzt TBH abstrakte Bereiche, mit denen die Entfernung dargestellt wird. **Benachbart**, **in der Nähe**, **abseits** und **entfernt**. Während eines Zugs können sich Charaktere irgendwo **in der Nähe** bewegen und ihre Handlung jederzeit während der Bewegung durchführen. Sie können ihre Handlung verfallen lassen, um sich **abseits** zu positionieren. Alles jenseits von **abseits** gilt als **entfernt** und würde drei Bewegungen erfordern.
+Anstatt genaue Werte nachzuhalten, benutzt TBH abstrakte Bereiche, mit denen die Entfernung dargestellt wird. **Schlagweite**, **Schrittweite**, **Schussweite** und **Sichtweite**. Während eines Zugs können sich Charaktere irgendwo in **Schrittweite** bewegen und ihre Handlung jederzeit während der Bewegung durchführen. Sie können ihre Handlung verfallen lassen, um sich in**Schussweite** zu positionieren. Alles jenseits von **Schussweite** gilt als in**Sichtweite** und würde drei Bewegungen erfordern.
```

Comment:
Nein, du hast recht. Ich habe einfach schlampig gearbeitet, mir ging es erst einmal darum, alle entsprechenden Reichweiten einzusetzen, um zu sehen, ob sich die Formulierungen in irgendeinem Kontext reiben...

Meiner Meinung nach funktioniert das grammatikalisch aber überall ganz manierlich...